src/GurobiSolverInterface.jl

```diff
@@ -25,14 +25,35 @@ function GurobiMathProgModel(;options...)
    return m
 end
 
+function GurobiMathProgModel(cloudhost::ASCIIString, password::ASCIIString; options...)
+   env = Env(cloudhost, password)
+   setparam!(env, "InfUnbdInfo", 1)
+   for (name,value) in options
+       setparam!(env, string(name), value)
+   end
+   m = GurobiMathProgModel(Model(env,""; finalize_env=true), :Con, false, Float64[], Float64[], nothing, nothing, nothing, nothing)
+   return m
+end
+
 
 immutable GurobiSolver <: AbstractMathProgSolver
     options
 end
+
+immutable GurobiCloudSolver <: AbstractMathProgSolver
```

Comment:
I would structure this just by adding options to the existing ``GurobiSolver`` object instead of creating a new solver type.